optaplanner-examples/src/main/java/org/optaplanner/examples/nurserostering/solver/move/factory/ShiftAssignmentPillarPartSwapMoveFactory.java

```diff
@@ -32,7 +32,7 @@
 import org.optaplanner.examples.nurserostering.domain.NurseRoster;
 import org.optaplanner.examples.nurserostering.domain.ShiftAssignment;
 import org.optaplanner.examples.nurserostering.domain.solver.ShiftAssignmentPinningFilter;
-import org.optaplanner.examples.nurserostering.score.drools.EmployeeWorkSequence;
+import org.optaplanner.examples.nurserostering.optional.score.EmployeeWorkSequence;
```

Comment:
Why does the move factory depend on a class that is used only by the DRL score calculation? Does it mean the move factory should be used only with that score calculation type?

Reply:
No, it just uses some code from that class. But I see your point - maybe the class should not be optional then. I'll see what I can do about this, but I definitely do not want to overengineer the package structure.

Reply:
Maybe we should move that class to some `common` package? Your call.